frontend/src/pages/LandlordPage.tsx

```diff
@@ -28,13 +31,63 @@ const reviews = [
 
 const LandlordPage = (): ReactElement => {
   const { landlordId } = useParams<Record<string, string | undefined>>();
+  const features = ['Parking', 'Heating', 'Trash removal', 'Snow plowing', 'Maintenance'];
+  const info = ['111 Dryden Rd', '151 Dryden Rd', '418 Eddy St'];
+  const phone = '555-555-5555';
+  const address = '119 S Cayuga St, Ithaca, NY 14850';
+  const [width, setWidth] = useState(window.innerWidth);
```

Comment:
We already have an `isMobile` util file [here](https://github.com/cornell-dti/cu-apts/blob/main/frontend/src/utils/isMobile.tsx) on the main branch. Maybe you can add a `getWidth` function here that returns the window width so we don't have to create new states every time.

(also if you could, also make both files in the `utils` folder end in `.ts` since they're both not React components, guess I forgot to review that part last time in Mena's PR)